### ISSUE TITLE
[Bug]: Image on a KButton does not react to the Padding

### ISSUE THREAD

**Issue by giduac** [Krypton-Suite/Standard-Toolkit#899]
**Is your feature request related to a problem? Please describe.**
No it's not really a problem.

**Describe the solution you'd like**
As you can see in the image the button-image (if left or right) is on the button edge.

**Describe alternatives you've considered**
Place the image a little more inwards

**Additional context**
![image](https://user-images.githubusercontent.com/96108132/212534659-3bb228d0-b7a8-45e8-a46d-4114586bf7f0.png)

**Comment by Smurf-IV** [comment]
@giduac What you are describing appears to be be a bug.
Can you indicate Krypton version and OS please?

Also using the DEvBuild of alpha and the tolkit examples in Windows 10 22H2 I get this:
![image](https://user-images.githubusercontent.com/2418812/212535044-d226f031-af5a-4dfa-8c21-b0feb41ea31f.png)

So have you set some padding values to be negative ?

**Comment by giduac** [comment]
Hi @Smurf-IV 

Image  placement:
`btnGenerate.StateCommon.Back.ImageStyle = PaletteImageStyle.CenterLeft;`

I'm using:
KT: 70.22.11.312
VS2022 17.4.3
W10 20H2 also on W11 21H2 (not connected to the OS version imho)

**Comment by giduac** [comment]
Tried the left padding:
-1 / 5 / 10 /50
Non of these values move the icon but the button text is moved to the right.

**Comment by giduac** [comment]
@Smurf-IV 
Will test it a little further this week with a separate test prog.
I'll let you know. 

Unless you have a solution of course... :)

**Comment by giduac** [comment]
Hi @Smurf-IV,

I removed the earlier messages. 

A new test has been done, my findings below.

- New & clean project (attached).
- TK versions tested: 80.22.12.356-beta / 70.22.11.312 / 60.22.2.32
- Framework versions tested: net48 / net60
- VStudio versions: 2019 / 2022 (no difference)
- Win10 20H2
-------------------------------------------------
* Buttons are added out of the box.
* Image is added (also in the project)
* Eventually apply corner rounding
* Applying Content Padding on StateCommon and/or StateNormal has no effect on the image only the button text.
* On a standard WinForms button the padding property does move the image inwards.


[899-IconPlacement.zip](https://github.com/Krypton-Suite/Standard-Toolkit/files/10423686/899-IconPlacement.zip)

**Comment by Smurf-IV** [comment]
@giduac 
Thanks for the Example. I was able to find out what had been done.
You had set the image into the Background drawing, and then set it's offset in that, that Padding of contents would not affect it.

So I moved the image into the "Content" settings and then set it's positioning characteristics (Near H, Center V), and then set an offset on those:
![image](https://user-images.githubusercontent.com/2418812/222949414-0312a654-ec03-4f80-b7f4-f35273d849e4.png)
